rakelib/artifacts.rake

```diff
@@ -43,17 +43,24 @@ namespace "artifact" do
     end.flatten.uniq
   end
 
+  task "use-core-gemfile" do
+    base_dir = "pkg/gemfile/defaults"
+    FileUtils.cp("#{base_dir}/Gemfile.original", "Gemfile")
+  end
+
   task "use-defaults-gemfile" do
-    FileUtils.cp("Gemfile.defaults", "Gemfile")
-    FileUtils.cp("Gemfile.jruby-1.9.lock.defaults", "Gemfile.jruby-1.9.lock")
+    base_dir = "pkg/gemfile/defaults"
+    FileUtils.cp("#{base_dir}/Gemfile.defaults", "Gemfile")
+    FileUtils.cp("#{base_dir}/Gemfile.jruby-1.9.lock.defaults", "Gemfile.jruby-1.9.lock")
   end
 
   task "freeze-defaults-gemfile" => ["bootstrap", "plugin:install-default"] do
-    FileUtils.cp("Gemfile", "Gemfile.defaults")
-    FileUtils.cp("Gemfile.jruby-1.9.lock", "Gemfile.jruby-1.9.lock.defaults")
+    base_dir = "pkg/gemfile/defaults"
```

Comment:
Also this base_dir var could be moved to a constant DEFAULT_GEMFILE_DIR or similar